gef.py

```diff
@@ -326,6 +355,16 @@ def wrapper(*args, **kwargs):
     return wrapper
 
 
+def print_highlight_table():
+    if not highlight_table:
+        return err("no matches found")
+
+    left_pad = max(map(len, highlight_table.keys()))
+    for match, color in sorted(highlight_table.items()):
+        print('{} | {}'.format(Color.colorify(match.ljust(left_pad), color),
+                               Color.colorify(color, color)))
+
+
```

Comment:
TL;DR : always a `return` statement

*Long version:*
Ok so I'm gonna be a giant prick once again, but I really think it's important to **always** add a `return` statement when the function returns, even when it returns nothing. The (My?) reason is that this code for instance is perfectly valid

```py
def foo(bar):
    global a, b
    a = 1
    b = bar
    return
    b = 2
```

Anybody who reads such code understands that `b = 2` is a glitch that should be removed, but without `return` it becomes totally unclear what was intended as a value for `b`. 
Doing code reviews, I have seen real life cases where it actually meant a difference, so we might as well use good practices 😄

Reply:
I agree with always returning, I wonder if this could be setup in the lint rules so it doesn't need to come up in PR's?  It may be easier to comment about, also including double quotes & unused imports as linting rules would be good too.

It sounds like this problem goes down to an issue of side-effects in a dynamic language with no return types and accidental indentation.  Tests should catch a lot of these issues if done properly.

There's something that looks wrong about a single `return` at the end of a python function, because it's unnecessary dead-code.  If you'd like to add a `return` everywhere explicitly, why not `return None`? it's the same as `return` but it has intention behind it.

Reply:
👍  on the automation part. We should tweak pylint do that labour.